objcgen/embedder.cs

```diff
@@ -388,6 +390,17 @@ public int Compile ()
 							common_options.Append ("-DTOKENLOOKUP ");
 						}
 					}
+
+					if (Extension)
+					{
+						common_options.Append("-fapplication-extension ");
```

Comment:
Style 1: single line if statements do not have braces (or if they do, the opening brace is at the end of the if line).
Style 2: space before opening parenthesis in method calls.